eframe/src/lib.rs

```diff
@@ -95,9 +100,44 @@ pub use web_sys;
 ///     eframe::start_web(canvas_id, Box::new(|cc| Box::new(MyEguiApp::new(cc))))
 /// }
 /// ```
+#[cfg(target_arch = "wasm32")]
+use arc_swap::ArcSwapOption;
+
+#[cfg(target_arch = "wasm32")]
+unsafe impl Send for AppRunner {}
+
+#[cfg(target_arch = "wasm32")]
+unsafe impl Sync for AppRunner {}
+
+#[cfg(target_arch = "wasm32")]
+static GLOBAL_HANDLE: ArcSwapOption<Mutex<AppRunner>> = ArcSwapOption::const_empty();
+
```

Comment:
You moved the docstring for `start_web` to now document `use arc_swap::ArcSwapOption;`.